src/ec/packages/ec-component-timeline/timeline.html.twig

```diff
@@ -0,0 +1,120 @@
+{% spaceless %}
+
+{#
+Parameters:
+  - "toggleCollapsed": (string) (default: 'Show %d more items')
+  - "toggleExpanded": (string) (default: 'Show %d items')
+  - "hide" (array): (default: undefined)
+    - "from": (integer) (default: undefined) Positive integer
+    - "to": (integer) (default: undefined) Negative integer
+  - "items" (array) (default: []):
+    - "id": (string) (default: '')
+    - "label": (string) (default: '')
+    - "title": (string) (default: '')
+    - "content": (block) (default: '')
+  - "icon_path" (string) (default: ''): file containing the svg icons
+  - "extra_classes" (string) (default: '')
+  - "extra_attributes" (array) (default: []): format:
+    - "name" (string) (default: '')
+    - "value" (string) (default: '')
+#}
+
+{# Internal properties #}
+
+{% set _css_class = 'ecl-timeline2' %}
+{% set _extra_attributes = '' %}
+{% set _items = items|default([]) %}
+{% set _icon_path = icon_path|default('') %}
+{% set _toggle_collapsed = toggleCollapsed|default('Show %d more items') %}
+{% set _toggle_expanded = toggleExpanded|default('Hide %d items') %}
+{% set _hide_from = hide.from|default(items|length) %}
+{% set _hide_to = hide.to|default(items|length) %}
+
+{% set display_amount = items|length %}
```

Comment:
all internal variables should be prepended with an underscore (sorry, it's an undocumented convention)